client/reader/discover/_style.scss

```diff
@@ -1,73 +1,89 @@
-.discover-attribution {
-	font-size: 13px;
-	font-family: $sans;
-	text-transform: lowercase;
-	color: $gray;
-	position: relative;
-	margin-top: 13px;
-	margin-bottom: 8px;
+.reader__card.card, .reader__full-post {
 
-	&.is-site {
-		margin-top: 15px;
-	}
+	.discover-attribution {
+		font-size: 13px;
+		font-family: $sans;
+		text-transform: lowercase;
+		color: $gray;
+		position: relative;
+		margin-top: 13px;
+		margin-bottom: 8px;
+
+		a {
+			color: $blue-medium;
+			text-decoration: none;
 
-	&.is-missing-avatar {
-		&.is-post {
-			padding-left: 26px;
+				&:hover {
+				color: $blue-light;
+			}
 		}
-	}
 
-	.gridicon {
-		height: 12px;
-		width: 12px;
-		display: inline-block;
-		fill: $white;
-		background: $gray;
-		padding: 4px;
-		border-radius: 100%;
-		transform: rotate(180deg);
-		position: absolute;
-			top: 0;
-			left: 0;
-	}
+		&.is-site {
+			margin-top: 15px;
+		}
 
-	.gravatar {
-		background: $gray-light;
-		position: relative;
-			top: 6px;
-		margin: 1px 6px 1px 0;
-	}
+		&.is-missing-avatar {
+			&.is-post {
+				padding-left: 26px;
```

Comment:
This `padding-left` adds indentation to some missing gravatars:

![screenshot 2016-07-29 11 32 30](https://cloud.githubusercontent.com/assets/4924246/17259406/27d805e2-5580-11e6-9b84-9769251142f3.png)